app/views/cp_mailer/contract_email.html.erb

```diff
@@ -4,7 +4,7 @@
     <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
   </head>
   <body>
-    <h1>Congratulations, <%= @offer[:applicant][:first_name] %> <%= @offer[:applicant][:last_name] %></h1>
+    <p>Dear <%= @offer[:applicant][:first_name] %> <%= @offer[:applicant][:last_name] %>,</p>
     <p>
       You have successfully been choosen to TA <%= @offer[:position] %>
```

Comment:
Could we change this to:

We would like to offer you a position as a TA for <%= @offer[:position] %>.

Please click the link below to view your offer and accept or reject it:
 <%= @url %>.